decentraland/explorer
# chore: black overlay background added

In order to distinguish better the pop up from the rest of the UI, an overlay black background was added.

**How to test?**
Enter as a guest player so the client recognize you as a new user. After completing the tutorial you should see the message of the day.
